modules/rosa-create-objects.adoc

```diff
@@ -109,78 +107,99 @@ Create a new cluster.
 .Syntax
 [source,terminal]
 ----
-$ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
+$ rosa create cluster --cluster-name=<cluster_name> [arguments]
 ----
 
 .Arguments
 [cols="30,70"]
 |===
 |Option |Definition
 
-|--cluster
-|Required: The name or ID (string) of the cluster. When used with the `create cluster` command, this argument is used to generate a sub-domain for your cluster on `openshiftapps.com`.
+a|--cluster-name <cluster_name>
+|Required. The name of the cluster. When used with the `create cluster` command, this argument is used to set the cluster name and to generate a sub-domain for your cluster on `openshiftapps.com`. The value for this argument must be unique within your organization.
 
-|--compute-machine-type
-|The instance type (string) for the compute nodes. Determines the amount of memory and vCPU that are allocated to each compute node.
+|--compute-machine-type <instance_type>
+|The instance type for compute nodes in the cluster. This determines the amount of memory and vCPU that is allocated to each compute node. For more information on valid instance types, see link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/policies-and-service-definition#rosa-sdpolicy-aws-instance-types_rosa-service-definition[AWS Instance types].
 
-|--compute-nodes
-|The number (integer) of worker nodes to provision per zone. Single-zone clusters require at least 2 nodes. Multi-zone clusters require at least 3 nodes. Default: `2` for single-az; `3` for multi-az
+|--compute-nodes n
+|The number of worker nodes to provision per zone. Single-zone clusters require at least 2 nodes. Multi-zone clusters require at least 3 nodes. Default: `2` for single-zone clusters; `3` for multi-zone clusters.
 
-|--controlplane-iam-role string
-|The Amazon Resource Name (ARN) of the IAM role that will be attached to control plane instances.
+|--controlplane-iam-role <arn>
+|The Amazon Resource Name (ARN) of the IAM role to attach to control plane instances.
 
 |--disable-scp-checks
 |Indicates whether cloud permission checks are disabled when attempting to install a cluster.
 
 |--dry-run
 |Simulates creating the cluster.
 
-|--enable-autoscaling
+|--enable-autoscaling <number_of_nodes>
```

Comment:
--enable-autoscaling is a flag which don't need the <number_of_nodes>. I will be used combining with max-replicas and min-replicas

Reply:
Fixed, thankyou!